### PR TITLE
Updating Dockerfile (pip install)

### PR DESCRIPTION
**Pull request by Draichi**
I think it's time to update Dockerfile. The latest version of sonnet is in conflict with something.

 I was able to run the project by adding the `dm-sonnet` stable version and including `tensorflow_probability` to the list.

Related issues:

- #28 
- #16 
- https://github.com/deepmind/graph_nets/issues/2

### CODE REVIEW

**Comment by Draichi** [comment]
I think this could be tested, 

I added `tensorflow_probability` because I was having this isse https://github.com/deepmind/scalable_agent/issues/28#issue-396244350.

and the bazel version because i was having this issue https://github.com/deepmind/scalable_agent/issues/16#issuecomment-422445031

File: Dockerfile [hunk 1]
```diff
@@ -80,4 +96,4 @@ CMD ["sh", "-c", "python experiment.py --total_environment_frames=10000 --datase
 # Docker commands:
 #   docker rm scalable_agent -v
 #   docker build -t scalable_agent .
-#   docker run --name scalable_agent scalable_agent
```
Comment (Draichi) [inline Dockerfile]: not necessary, i dont even know what changed here

**Comment by lespeholt** [comment]
Seems like both of those issues have been fixed.

**Comment by lespeholt** [comment]
Happy to accept a change to use dm-sonnet==1.23

**Comment by Draichi** [comment]
Ok brother, updated on https://github.com/deepmind/scalable_agent/pull/29/commits/44626b47ec6839fa911aaa36474f196f5063ed05

**Comment by lespeholt** [comment]
Thanks for fixing it

**Pull request merged**